appengine/datastore/README.md

```diff
@@ -6,14 +6,41 @@ from [Google App Engine standard environment][ae-docs].
 [java-datastore]: https://cloud.google.com/appengine/docs/java/datastore/
 [ae-docs]: https://cloud.google.com/appengine/docs/java/
 
+
+## Running locally
+
+This example uses the
+[Maven gcloud plugin](https://cloud.google.com/appengine/docs/java/managed-vms/maven).
+To run this sample locally:
+
+    $ mvn appengine:devserver
+
+To see the results of the sample application, open
+[localhost:8080](http://localhost:8080) in a web browser.
+
+
+## Deploying
+
+In the following command, replace YOUR-PROJECT-ID with your
+[Google Cloud Project ID](https://developers.google.com/console/help/new/#projectnumber)
+and SOME-VERSION with a valid version number.
+
+    $ mvn appengine:update -Dappengine.appId=YOUR-PROJECT-ID -Dappengine.version=SOME-VERSION
+
+
 ## Setup
+
+To save your project settings so that you don't need to enter the
+ parameters, you can:
+
```

Comment:
Can't we just tell them to run it w/ mvn command line args instead of editing below.